ansible/advanced.md

```diff
@@ -9,26 +9,62 @@ has_toc: false
 
 ## Advanced configuration
 
-### ii leader mode
-
-Ansible can be used as a leader device for the [ii](/docs/modular/ii)
-bus, controlling one or more follower modules using trigger and CV
-information sent from Ansible. Since Ansible is not wired to supply
-power to the ii bus, another leader module or a powered busboard is
-required to use Ansible as a leader. Having [Crow](/docs/crow) or
-[Teletype](../teletype/) connected to the same ii bus is sufficient.
-
-When Ansible is acting as a ii leader, it can no longer receive ii
-commands from other leaders like Teletype or Crow. Note also that
-sending messages from multiple leader devices simultaneously on the
-same ii bus does work in some tested scenarios, but may increase the
-risk of buggy behavior including modules locking up and needing a
-power cycle.
-
-Currently, the only interface for turning on leader mode uses a grid.
-In any grid application, go to the preset page by pressing the button
-next to the USB port. Follower device toggles are arranged vertically,
-just left of center. From top to bottom:
+  ###  what is i2c/ii?
+
+    i2c, or [ii](/docs/modular/ii), is a flexible communication protocol which
+    allows modules to send commands to each other digitally, which opens up
+    possibilities that patch cables can’t facilitate. This digital networking is
+    described as a ‘bus’.
+
+    The i2c bus consists of 3 lines - ground (GND), data (SDA) and clock (SCL).
```

Comment:
This general discussion of i2c is useful, this maybe should go on the /docs/modular/ii page since it's applicable to a number of modules rather than in the ansible docs.

Reply:
@csboling , lol omg i totally forgot that existed. okay, yeah, i do think overhauling [this page](http://monome.org/docs/modular/ii) with the extended description and image would be good -- but more importantly, this needs to be surfaced on the sidebar 😅